### PR TITLE
[Merged by Bors] - fix: set variable value (COR-3108)

### PR DESCRIPTION
**Pull request by z4o4z**

### CODE REVIEW

**Comment by z4o4z** [comment]
* **#950** <a href="https://app.graphite.dev/github/pr/voiceflow/general-runtime/950?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @z4o4z and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: tests/lib/services/runtime/handlers/setV3/lib/evaluateValue.unit.ts [hunk 1]
```diff
@@ -34,7 +55,27 @@ describe('evaluateValue', () => {
...
+          value: '{variable3}',
+          label: 'Set Expression 2',
+          variable: 'variable3',
+        },
+        variables,
+        logToPrototype
+      );
+
+      expect(assignmentMap).to.eql({
+        variable3: [1, 2, 3],
```
Comment (zhihil) [inline tests/lib/services/runtime/handlers/setV3/lib/evaluateValue.unit.ts:78]: We should add a case for object as well. Zoran will likely encounter an issue with `object` as well.

Comment (z4o4z) [reply tests/lib/services/runtime/handlers/setV3/lib/evaluateValue.unit.ts:78]: why do we need a unit test for an object? Array is an object...

**Review by zhihil — APPROVED** [review]

**Review by z4o4z — COMMENTED** [review]
bors r+

**Review by JoshTheDeveloper — APPROVED** [review]

**Pull request closed without merging**